### PR TITLE
Fix Payloads/Payload CLI output

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
`Payloads` fields are displayed properly.
Was: `Input:{Payloads:[len=1]}`
Now: `Input:{"Temporal"}`

**Why?**
Fix the bug.

**How did you test it?**
`./tctl wf show -w hello_world_workflowID`

**Potential risks**
Some CLI commands will crash or produce wrong output.

### CODE REVIEW

File: tools/cli/util.go [hunk 1]
```diff
@@ -472,12 +469,12 @@ func getEventAttributes(e *eventpb.HistoryEvent) interface{} {
 }
 
 func isAttributeName(name string) bool {
-	for _, eventTypeName := range eventpb.EventType_name {
-		if name == eventTypeName+"EventAttributes" {
-			return true
-		}
+	eventType := strings.TrimSuffix(name, "EventAttributes")
 
+	if _, ok := eventpb.EventType_value[eventType]; ok {
```
Comment (alexshtin) [inline tools/cli/util.go:474]: ~41 times faster. And we call this for every field.

File: tools/cli/util.go [hunk 2]
```diff
@@ -909,3 +906,31 @@ func prompt(msg string, autoConfirm bool) {
...
+}
+
+func payloadToString(p *commonpb.Payload) string {
+	if p == nil {
+		return "<nil>"
+	}
+	var data string
+	err := payload.Decode(p, &data)
+	if err == nil {
+		return data
```
Comment (shawnhathaway) [inline tools/cli/util.go:932]: Maybe return constant here like `DECODING_ERROR`

File: tools/cli/util.go [hunk 3]
```diff
@@ -909,3 +906,31 @@ func prompt(msg string, autoConfirm bool) {
...
+	} else {
+		return string(p.GetData())
```
Comment (shawnhathaway) [inline tools/cli/util.go:934]: What does this output?

**Comment by markmark206** [comment]
yay!! i was just debugging something that involved `Details:{Payloads:[len=1]}`, this is very useful!

```
bash-5.0# tctl --ns benchtest wf show -w Kvkx
  1  WorkflowExecutionStarted  {WorkflowType:{Name:basic-driver},
                                ParentInitiatedEventId:0, TaskList:{Name:samples_tl,
                                Kind:Normal}, Input:{Payloads:[len=1]},
                                WorkflowExecutionTimeoutSeconds:259200,
                                WorkflowRunTimeoutSeconds:259200,
                                WorkflowTaskTimeoutSeconds:5, Initiator:Decider,
                                OriginalExecutionRunId:49431e1a-025e-4c0d-9425-fab7ed88dc8d,
                                Identity:tctl@168538bfd6c5,
                                FirstExecutionRunId:49431e1a-025e-4c0d-9425-fab7ed88dc8d,
                                Attempt:0,
                                WorkflowExecutionExpirationTimestamp:1589760357832000000,
                                FirstDecisionTaskBackoffSeconds:0}
  2  DecisionTaskScheduled     {TaskList:{Name:samples_tl,
                                Kind:Normal},
                                StartToCloseTimeoutSeconds:5,
                                Attempt:0}
  3  DecisionTaskStarted       {ScheduledEventId:2, Identity:1@0fa00ab1ff09@,
                                RequestId:00df41db-a40f-46c3-ad38-5035eb80fcf5}
  4  DecisionTaskCompleted     {ScheduledEventId:2, StartedEventId:3,
                                Identity:1@0fa00ab1ff09@,
                                BinaryChecksum:f45ae389ecfa34f71c96581996b76a7b}
  5  WorkflowExecutionFailed   {Reason:temporalInternal:Generic,
                                Details:{Payloads:[len=1]},
                                DecisionTaskCompletedEventId:4}
```

File: tools/cli/util.go (hunk 3 above)
Comment (alexshtin) [reply tools/cli/util.go:934]: It is a very good question. Actually with this implementation only `string` is displayed properly. I fixed it in #377.

**Pull request merged**